From a9059be749b6043d6cbc5b79652e8a4adda8994e Mon Sep 17 00:00:00 2001
From: Hui Wang <hui.wang@canonical.com>
Date: Tue, 24 Mar 2015 10:29:15 +0100
Subject: [PATCH 308/311] daemon-conf: enable the lfe remixing by default

Since we have a workable lfe filter, it is time to enable the lfe
remixing by default.

Signed-off-by: Hui Wang <hui.wang@canonical.com>
---
 man/pulse-daemon.conf.5.xml.in | 2 +-
 src/daemon/daemon-conf.c       | 2 +-
 src/daemon/daemon.conf.in      | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

Index: pulseaudio/man/pulse-daemon.conf.5.xml.in
===================================================================
--- pulseaudio.orig/man/pulse-daemon.conf.5.xml.in	2015-05-13 15:00:23.030148165 +0800
+++ pulseaudio/man/pulse-daemon.conf.5.xml.in	2015-05-13 15:00:23.026148165 +0800
@@ -117,7 +117,7 @@
       channel is available as well. If no input LFE channel is
       available the output LFE channel will always be 0. If no output
       LFE channel is available the signal on the input LFE channel
-      will be ignored. Defaults to <opt>no</opt>.</p>
+      will be ignored. Defaults to <opt>yes</opt>.</p>
     </option>
 
     <option>
Index: pulseaudio/src/daemon/daemon-conf.c
===================================================================
--- pulseaudio.orig/src/daemon/daemon-conf.c	2015-05-13 15:00:23.030148165 +0800
+++ pulseaudio/src/daemon/daemon-conf.c	2015-05-13 15:00:23.026148165 +0800
@@ -82,7 +82,7 @@
     .log_time = false,
     .resample_method = PA_RESAMPLER_AUTO,
     .disable_remixing = false,
-    .disable_lfe_remixing = true,
+    .disable_lfe_remixing = false,
     .lfe_crossover_freq = 120,
     .config_file = NULL,
     .use_pid_file = true,
Index: pulseaudio/src/daemon/daemon.conf.in
===================================================================
--- pulseaudio.orig/src/daemon/daemon.conf.in	2015-05-13 15:00:23.030148165 +0800
+++ pulseaudio/src/daemon/daemon.conf.in	2015-05-13 15:00:23.026148165 +0800
@@ -54,7 +54,7 @@
 
 ; resample-method = speex-float-1
 ; enable-remixing = yes
-; enable-lfe-remixing = no
+; enable-lfe-remixing = yes
 ; lfe-crossover-freq = 120
 
 flat-volumes = no
